traffic_ops/traffic_ops_golang/cachegroup/cachegroups.go

```diff
@@ -427,6 +432,7 @@ func (cg *TOCacheGroup) Read() ([]interface{}, error, error, int) {
 		"name":      dbhelpers.WhereColumnInfo{"cachegroup.name", nil},
 		"shortName": dbhelpers.WhereColumnInfo{"cachegroup.short_name", nil},
 		"type":      dbhelpers.WhereColumnInfo{"cachegroup.type", nil},
+		"topology":  dbhelpers.WhereColumnInfo{"topology_cachegroup.topology", nil},
```

Comment:
This can be

```go
		"topology":  {"topology_cachegroup.topology", nil},
```

or

```go
		"topology":  {Column: "topology_cachegroup.topology"},
```

Reply:
chose 1st option in 59d6fb1005576218c08246171707843ba05e3e6f